test/stdlib/Dispatch.swift

```diff
@@ -119,6 +115,17 @@ DispatchAPI.test("DispatchTime comparisons") {
     }
 }
 
+DispatchAPI.test("DispatchTime.create") {
+	var time = DispatchTime(uptimeNanoseconds: 0)
+	expectEqual(time.uptimeNanoseconds, 0)
+
+	time = DispatchTime(uptimeNanoseconds: 15 * NSEC_PER_SEC)
+	expectEqual(time.uptimeNanoseconds, 15 * NSEC_PER_SEC)
+
+	time = DispatchTime(uptimeNanoseconds: UInt64.max)
+	expectEqual(time.uptimeNanoseconds, UInt64.max)
+}
+
```

Comment:
I don't think any of these exercise the overflow paths. You should add values like `UInt64.max-1`.

Reply:
That's a good point and when I tried some of the larger values, it turns out that the forced ceiling of UInt64.max for the multiplication produces some odd results. I am going to revisit that.